RecoEgamma/EgammaPhotonProducers/python/ootPhotons_cff.py

```diff
@@ -1,32 +1,48 @@
 import FWCore.ParameterSet.Config as cms
 from RecoEgamma.EgammaPhotonProducers.gedPhotons_cfi import gedPhotons as _gedPhotons
 
-ootPhotons = _gedPhotons.clone(
+ootPhotonsTmp = _gedPhotons.clone(
     photonProducer = 'ootPhotonCore',
     candidateP4type = "fromEcalEnergy",
     reconstructionStep = "oot",
     pfEgammaCandidates = "",
     valueMapPhotons = ""
     )
+del ootPhotonsTmp.regressionConfig
+
+ootPhotons = _gedPhotons.clone(
+    photonProducer = 'ootPhotonsTmp',
+    candidateP4type = "fromEcalEnergy",
```

Comment:
@Sam-Harper , checking https://github.com/Sam-Harper/cmssw/blob/6e6a25fd65e53bbf5a43d99fecab8a2f46712896/RecoEgamma/EgammaPhotonProducers/src/GEDPhotonProducer.cc#L841-L843, this means that the regression is effectively skipped in the final step for ootPhotons, correct?  

I guess the nice thing here is that, now the symmetry between ged and oot photons is closer with a tmp and final step for both, even if nothing really happens in the final step for oot photons.

Reply:
Yes, the intention was to skip the regression for OOT photons.